example/src/components/stacks/SendSip10/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button, Card, Stack, TextInput } from '@mantine/core';
+import {
+  // bufferCV,
+  cvToString,
+  noneCV,
+  // someCV,
+  standardPrincipalCV,
+  uintCV,
+} from '@stacks/transactions';
+import { ChangeEventHandler, useState } from 'react';
+import Wallet, { Address, BitcoinNetworkType } from 'sats-connect';
+import { Success } from '../../../App.styles';
+
+// TODO use emptry strings once done testing
+const formInitialState = {
+  amount: '100000000', // 100LEO
+  contract: 'SP1AY6K3PQV5MRT6R4S671NWW2FRVPKM0BR162CT6.leo-token', // LEO token contract
+  address: 'SP313FW47A0XR7HCBFQ0ZZHS47Q265AEBMPK1GD4N', // account 2
+  memo: '',
+};
+
+const formInputs: {
+  field: keyof typeof formInitialState;
+  label: string;
+  type: 'text' | 'number';
+}[] = [
+  {
+    field: 'contract',
+    label: 'Contract',
+    type: 'text',
+  },
+  {
+    field: 'amount',
+    label: 'Amount',
+    type: 'number',
+  },
+  {
+    field: 'address',
+    label: 'Address',
+    type: 'text',
+  },
+  {
+    field: 'memo',
+    label: 'Memo (optional)',
+    type: 'text',
+  },
+];
+
+export const SendSip10 = ({
+  network,
+  stxAddressInfo,
+}: {
+  network: BitcoinNetworkType;
+  stxAddressInfo: Address[];
+}) => {
+  const [form, setForm] = useState(formInitialState);
+  const [txnId, setTxnId] = useState('');
+
+  const canSubmit = form.amount && form.address && form.contract;
+
+  const getChangeFormHandler =
+    (fieldName: keyof typeof formInitialState): ChangeEventHandler<HTMLInputElement> =>
+    (e) => {
+      setForm((prevForm) => ({ ...prevForm, [fieldName]: e.target.value }));
+    };
+
+  const onClick = () => {
+    (async () => {
+      const response = await Wallet.request('stx_callContract', {
+        contract: form.contract,
+        functionName: 'transfer',
+        arguments: [
+          uintCV(+form.amount),
+          standardPrincipalCV(stxAddressInfo?.[0].address),
+          standardPrincipalCV(form.address),
+          noneCV(),
+          //form.memo ? someCV(bufferCV(Buffer.from(form.memo))) : noneCV(),
+        ].map((arg) => cvToString(arg)),
+      });
```

Comment:
@m-aboelenein @aryzing any ideas on whether there's something causing the error here?

Reply:
In the map, try with `cvToHex()`

Reply:
nice! thank you, i'll try that and make a note to update the docs https://docs.xverse.app/sats-connect/stacks-methods/stx_callcontract

actually a lot of the stx docs seem to be incomplete